app/mailers/contact_mailer.rb

```diff
@@ -0,0 +1,23 @@
+class ContactMailer < ApplicationMailer
+  ADMIN_EMAIL = "yuyueureka157@gmail.com"
+  SAMPLE_EMAIL = "test@sample.com"
+
+  def user_email(contact)
+    @contact = contact
+    @name = contact[:name].presence || contact[:email]
+    subject = "「やんばるエキスパートwebデザイン講座」お問い合わせ受付"
+
+    mail(to: contact[:email], subject: subject)
+  end
+
+  def admin_email(contact)
+    @contact = contact
+    @name = contact[:name].presence || contact[:email]
+    subject = "「やんばるエキスパートwebデザイン講座」お問い合わせ受付"
+    receiver = ADMIN_EMAIL
+    if Rails.env.development?
+      receiver = SAMPLE_EMAIL
+    end
```

Comment:
こちらの方がまだ分かりやすいと思います。

```suggestion
    receiver = Rails.env.development? ? SAMPLE_EMAIL : ADMIN_EMAIL
```

もしくはこちらでしょうか。

```rb
    receiver = if Rails.env.development?
                 SAMPLE_EMAIL
               else
                 ADMIN_EMAIL
               end
```